### PR TITLE
Update jwt section

### PR DESCRIPTION
**Pull request by korycins**
Update docs for changes from PR: https://github.com/mirumee/saleor/pull/5734

### CODE REVIEW

File: docs/developer/running-saleor/configuration.mdx [hunk 1]
```diff
@@ -185,6 +185,19 @@ Controls [Django’s secret key](https://docs.djangoproject.com/en/3.0/ref/setti
...
 Controls the production assets mount path. Defaults to `/static/`.
 
+### `JWT_EXPIRATION_DELTA`
+
+Controls the time which defines how long the JWT authentication token will be valid. The value should be provided in seconds. Defaults to 5 minutes.
+
+
+### `JWT_REFRESH_EXPIRATION_DELTA`
+
+Controls the time which defines how long the JWT refresh token will be valid. The value should be provided in seconds. Default to 30 days.
```
Comment (NyanKiyoshi) [inline docs/developer/running-saleor/configuration.mdx:195]: Not seconds, but `timedelta` objects

Comment (korycins) [reply docs/developer/running-saleor/configuration.mdx:195]: `seconds`

Comment (korycins) [reply docs/developer/running-saleor/configuration.mdx:195]:
```
JWT_EXPIRATION_DELTA = timedelta(
    seconds=os.environ.get("JWT_EXPIRATION_DELTA", 5 * 60)  # type: ignore
)
```

Comment (NyanKiyoshi) [reply docs/developer/running-saleor/configuration.mdx:195]: Ah nevermind, I see we changed it in the PR https://github.com/mirumee/saleor/blob/877810e83da772569f51ac5650edd11ceed4f629/saleor/settings.py#L552-L561

Comment (korycins) [reply docs/developer/running-saleor/configuration.mdx:195]: Yeap, we drop the graphql_jwt package.

File: docs/developer/running-saleor/configuration.mdx [hunk 2]
```diff
@@ -185,6 +185,19 @@ Controls [Django’s secret key](https://docs.djangoproject.com/en/3.0/ref/setti
...
+
+### `JWT_SECRET`
+
+Controls the key used to encode/decode JWT token.
```
Comment (maarcingebala) [inline docs/developer/running-saleor/configuration.mdx:199]: We should also say that it defaults to `SECRET_KEY` with a link to this value's description: https://docs.saleor.io/docs/developer/running-saleor/configuration#secret_key

Comment (korycins) [reply docs/developer/running-saleor/configuration.mdx:199]: Yes! I forgot about this. Thank you 👮  !

File: docs/developer/users.mdx [hunk 3]
```diff
@@ -254,41 +271,58 @@ curl \
...
-### Refreshing a token
+### Refreshing an access token
+
+The `tokenRefresh` mutation will generate a new access token when provided with a valid refresh token and a matching CSRF token.
+If the refresh token is not provided as an argument, the code will try to read it from a cookie set by the `tokenCreate` mutation.
+
+The mutation takes the following input fields:
+
+- `refreshToken`: the refresh token, if not specified will be read from browser cookies.
+- `csrfToken`: the CSRF token used to protect against cross-site resource forgery attacks.
```
Comment (patrys) [inline docs/developer/users.mdx:315]: We need to update this section to reflect that only one of those is required. We should also provide separate examples for those two cases.

Comment (korycins) [reply docs/developer/users.mdx:315]: Yes, sorry. I forgot about this.

File: docs/developer/users.mdx [hunk 4]
```diff
@@ -254,41 +271,59 @@ curl \
...
+#### Refreshing with refresh token as a browser cookie
+
+The mutation takes the following input fields:
+
+- `csrfToken`: the CSRF token used to protect against cross-site resource forgery attacks.
 
 ```graphql {2}
 mutation {
-  tokenRefresh(token: "<your-token>") {
+  tokenRefresh( csrfToken: "<your-csrf-token>") {
```
Comment (patrys) [inline docs/developer/users.mdx:322]:
```suggestion
  tokenRefresh(csrfToken: "<your-csrf-token>") {
```

**Comment by korycins** [comment]
@patrys  I think that we can merge it. The code for JWT token is already merged

**Pull request merged**